### DISCUSSION TITLE
C# read log  Emgu.CV.runtime.windows

### DISCUSSION THREAD

**Discussion by barabba2005** [emgucv/emgucv#625] [Q&A]
Hi all

I'm writing a little program for face detection.
I’m would know how read the logs from Emgu.CV.runtime.windows Vers. 4.5.3.472.

Thanks in advance

**Comment by barabba2005** [comment]
can someone help me?
Thanks

**Comment by emgucv** [comment]
This may help:
https://stackoverflow.com/questions/54828885/how-to-enable-logging-for-opencv

Open CV's log is written to standard streams. On windows, you may need to start your program from command line, logs will be printed out on console.
